From e9788fdcf00b058f4404b0b7164895f7ad348a88 Mon Sep 17 00:00:00 2001
From: Alex Kiernan <alex.kiernan@hivehome.com>
Date: Sat, 30 Jun 2018 17:03:51 +0000
Subject: [PATCH] boot: Use emergency.target, not emergency.service

Follow systemd units in using emergency.target, not emergency.service
(which is the sole unit, by default, in emergency.target) so we can
easily reconfigure the units which are actived when entering
emergency mode.

Signed-off-by: Alex Kiernan <alex.kiernan@gmail.com>
---
 src/boot/ostree-prepare-root.service | 2 +-
 src/boot/ostree-remount.service      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/boot/ostree-prepare-root.service b/src/boot/ostree-prepare-root.service
index e0659d10..52cf8863 100644
--- a/src/boot/ostree-prepare-root.service
+++ b/src/boot/ostree-prepare-root.service
@@ -20,7 +20,7 @@ Description=OSTree Prepare OS/
 DefaultDependencies=no
 ConditionKernelCommandLine=ostree
 ConditionPathExists=/etc/initrd-release
-OnFailure=emergency.service
+OnFailure=emergency.target
 After=initrd-switch-root.target
 Before=initrd-switch-root.service
 Before=plymouth-switch-root.service
diff --git a/src/boot/ostree-remount.service b/src/boot/ostree-remount.service
index 8439b495..68209f96 100644
--- a/src/boot/ostree-remount.service
+++ b/src/boot/ostree-remount.service
@@ -19,7 +19,7 @@
 Description=OSTree Remount OS/ bind mounts
 DefaultDependencies=no
 ConditionKernelCommandLine=ostree
-OnFailure=emergency.service
+OnFailure=emergency.target
 Conflicts=umount.target
 After=-.mount
 After=systemd-remount-fs.service
-- 
2.17.1

